src/staff_augmentation/components/HowItWorks/styles.module.scss

```diff
@@ -0,0 +1,186 @@
+@import '/src/styles/theme.scss';
+@import '/src/styles/media_queries.scss';
+
+.howItWorksSection {
+  display: flex;
+  flex-direction: column;
+  margin-top: -130px;
+  padding-top: 175px;
+  padding-bottom: 50px;
+  background: rgba($color: $buttonBackground, $alpha: 1.0);
+  color: rgba($color: #FFFFFF, $alpha: 1.0);
+
+  @include for-desktop-up {
+    padding-top: 250px;
+    padding-bottom: 150px;
+  }
+}
+
+.sectionHeader {
+  text-align: center;
+  margin-bottom: 20px;
+
+  @include for-desktop-up {
+    margin-bottom: 130px;
+  }
+}
+
+.subheadingText {
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  margin-bottom: 17px;
+}
+
+.titleText {
+  > span {
+    text-decoration: underline;
+  }
+}
+
+.sectionContent {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
+  margin: 0;
+  padding: 0px 50px 50px;
+
+  @include for-desktop-up {
+    flex-direction: row;
+  }
+}
+
+.contentColumn {
+  flex-grow: 1;
+  width: 100%;
+  text-align: center;
+  margin: auto;
+
+  @include for-desktop-up {
+    margin: 0;
+  }
+}
+
+.arrowContainer {
+  width: 100%;
+  position: relative;
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.arrowContainer::before {
+  @include for-desktop-up {
+    content: '';
+    height: 6px;
+    left: 0;
+    background: rgba($color: #FFFFFF, $alpha: 1.0);
+  }
+}
+
+.arrowContainer::after {
+  @include for-desktop-up { 
+    content: '';
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    height: 6px;
+    left: calc(50% + 25px + 30px);
+    right: 0;
+    background: rgba($color: #FFFFFF, $alpha: 1.0);
+  }
+}
+
+.arrowTip {
+  @include for-desktop-up { 
+    width: 19px;
+    height: 19px;
+    border-right: 6px rgba($color: #FFFFFF, $alpha: 1.0) solid;
+    border-bottom: 6px rgba($color: #FFFFFF, $alpha: 1.0) solid;
+    transform: translateY(-50%) rotate(-45deg) !important;
```

Comment:
Is the `!important` really necessary? You should reorder your class definitions instead of using this rule.